app/views/WhatDoYouWantToDoTileView.scala.html

```diff
@@ -41,22 +41,22 @@
   }
 }
 
+
 @mainTemplate(
     title = messages("your.paye.income.tax.overview"),
     authedUser = Some(user),
-    optimizelyAudience = optimizelyAudience,
-    showUrBannerOverride = true
```

Comment:
This has removed the UR banner from the main page, is this intended?

Reply:
good spot - we need  the banner - however just noticed we dont have this feature in the new main template...

Reply:
Going to be resolved as part of another ticket